### ISSUE TITLE
[subplat] Internal Server Error for PayPal privacy link

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#10164]
**Affected browser versions:**

* All Chrome;

**Affected FxA versions:**

* 1.213.1;

**Prerequisites:**

* Have a VPN set to Germany location;
* Have an FxA account with no active subscription;

**Steps to reproduce:**
1. Open VPN homepage https://www-dev.allizom.org/de/products/vpn/;
2. Click “Get Mozilla VPN” and choose any plan;
3. Click on the “PayPal privacy policy” link

**Expected result:**

* The Paypal PP page is displayed.

**Actual result:**

* Internal server error page is displayed.

**Notes:**

* reproducible from all subscriptions pages;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-3862)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/secure/attachment/16014/Untitled.png">Untitled.png</a>

**Comment by biancadanforth** [comment]
Hmm that's interesting, as I [found that earlier](https://github.com/mozilla/fxa/pull/10074#discussion_r687081916), and I thought it was updated.

Also, at that time I had similarly gotten an Internal Server Error, but when I tried it just now, I got redirected to https://www.paypal.com/webapps/mpp/ua/privacy-full which is the correct URL per [this comment](https://mozilla-hub.atlassian.net/browse/FXA-3715?focusedCommentId=441990).

I'm going to guess that, because we pass the updated URL into a localized message, even if the user-visible text didn't change, we still need to update the localization ID for the href value of the link to be updated.
